zerver/lib/transfer.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from django.conf import settings
+
+from zerver.models import UserProfile
+from zerver.lib.avatar_hash import user_avatar_path
+from zerver.lib.upload import read_local_file, S3UploadBackend
+
+s3backend = S3UploadBackend()
+
+def transfer_avatars_to_s3() -> None:
+    user_profiles = UserProfile.objects.all()
+    for user_profile in user_profiles:
+        avatar_path = user_avatar_path(user_profile)
+        print(avatar_path)
+        file_path = os.path.join(settings.LOCAL_UPLOADS_DIR, "avatars", avatar_path) + ".original"
+        try:
+            with open(file_path, 'rb') as f:
+                s3backend.upload_avatar_image(f, user_profile, user_profile)
+                print("Uploaded avatar for {} in {}".format(user_profile.email, user_profile.realm.name))
+        except FileNotFoundError:
+            print("Missing avatar for {} in {}".format(user_profile.email, user_profile.realm.name))
```

Comment:
This looks reasonable.  I think ideally we'd upload not just the `.original` file (if any), but also the other file sizes.

Reply:
@timabbott Yup. `upload_avatar_image` will upload all the 3 versions :)